src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -22,8 +22,9 @@ public Navigate(WebDriver driver) {
   public Navigate toPage(String pageName) {
     String host = UrlBuilder.getHostForWiki();
     String cacheBuster = pageName.equals("") || pageName.equals("/") ? "" : "?cb=" + DateTime.now().getMillis();
+    String abGroup = pageName.equals("") || pageName.equals("/") ? "" : defaultTestingGroup;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "/" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)